### PR TITLE
Create UI for profile page

### PR DESCRIPTION
**Pull request by Licheng-Wu**
Currently have minimal functionality, still trying to understand Typescript

### CODE REVIEW

**Comment by cjunxiang** [comment]
thanks for the effort! will look through very soon

File: frontend/src/routes/Routes.tsx [hunk 1]
```diff
@@ -9,11 +9,13 @@ export enum PATHS {
   // AUTHENTICATION
   LOGIN_PAGE = '/login',
   SIGNUP_PAGE = '/signup',
+  PROFILE_PAGE = '/profile',
```
Comment (cjunxiang) [inline frontend/src/routes/Routes.tsx:12]: existing conflict with main-branch. Resolve it before merging!

File: frontend/src/routes/Profile/index.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  width: 45vw;
+  vertical-align: middle;
+  padding-left: 3vw;
+`
+
+export default class Profile extends React.Component {
+  render() {
+    return (
+      <MainContainer>
+        <div style={{ marginLeft: '30vw' }}>TOPNAVBAR HERE</div>
```
Comment (cjunxiang) [inline frontend/src/routes/Profile/index.tsx:49]: after merging, can import top-nav-bar le.

File: frontend/src/routes/Profile/Components/PersonalInfoContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import styled from 'styled-components'
+import 'antd/dist/antd.css'
+
+const NameParagraph = styled.p`
+  font-family: Inter;
+  font-size: 24px;
+  font-weight: bold;
+`
+const TelegramParagraph = styled.p`
+  font-family: Inter;
```
Comment (cjunxiang) [inline frontend/src/routes/Profile/Components/PersonalInfoContainer.tsx:11]: next time we will shift font-family into root style file.

File: frontend/src/routes/Profile/Components/PersonalInfoContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      telegram={personalInfo.telegram}
+      block={personalInfo.block}
+      bio={personalInfo.bio}
+    ></PersonalInfoSecondaryContainer>
+  )
+}
+
+const PersonalInfoSecondaryContainer: React.FC<PersonalInfo> = (personalInfo: PersonalInfo) => {
+  return (
+    <>
```
Comment (cjunxiang) [inline frontend/src/routes/Profile/Components/PersonalInfoContainer.tsx:52]: Try to style the overall container with drop shadow, border-radius etc to fit figma styling!

**Review by cjunxiang — CHANGES_REQUESTED** [review]
Looks cool generally! -> resolve commits -> import top nav bar -> improve styling (or just take the css code from figma) -> ask for review again!

**Comment by Licheng-Wu** [comment]
Made changes according to comments, but idk how vercel deployment failed

**Pull request closed without merging**